README.md

```diff
@@ -102,6 +102,13 @@ WebAuthn.configure do |config|
   # Relying Party name for display purposes
   config.rp_name = "Example Inc."
 
+  # Optionally configure a client timeout hint, in milliseconds.
```

Comment:
This timeout is for waiting user input, with expected values of 15 or 30 seconds and up.
So I think seconds it's enough granularity for this type of timeout. I don't see milliseconds providing any extra value to the user.

Reply:
The spec states that the value should be in milliseconds. I agree that it makes more sense to have the value set in seconds. Nonetheless, I think there could be some confusion diverging from the spec.

Reply:
Fair enough.
Let's keep milliseconds :-)